### ISSUE TITLE
Screen reader does not announce the error information appears while editing the code elements.

### ISSUE THREAD

**Issue by msftedad** [swagger-api/swagger-ui#10443]
### Q&A (please complete the following information)

1.  OS: Windows 11
2. Browsers: Microsoft Edge (Version 135.0.3179.98) and Google Chrome (Version 135.0.7049.115)
3. Versions: Version 135.0.3179.98 and Version 135.0.7049.115
4. Method of installation: dist
5. Swagger-Editor version: 4.0.3
6. Swagger/OpenAPI Version: Swagger 2.0

### Content & configuration

Example Swagger/OpenAPI definition:
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Nintex%20Workflow/apiDefinition.swagger.json

Swagger-UI configuration options: 
```js
SwaggerUI({
  // your config options here
})
```

```
?yourQueryStringConfig
```

### To reproduce...

1. Open URL: https://editor.swagger.io/
2. Navigate inside the "Code editor" and enter any value.
3. Verify whether the screen reader announces the error information or not.

**Actual Result**
Screen reader does not announce the error information appears while editing the code elements.

### Expected behavior
Screen reader should announce the error information appears while editing the code element
Example: Screen reader should announce error information as ' Errors, Resolver error Cannot set properties of undefined (setting 'swagger')' while editing the code elements.

### Screenshots

https://github.com/user-attachments/assets/dd4192e9-c1e4-4235-87ef-0bb504a79adb
![Image](https://github.com/user-attachments/assets/aadabe57-7432-48cd-aa5f-b98fba43e0c8)

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
